### ISSUE TITLE
pymysql & mysqldb error report

### ISSUE THREAD

**Issue by fizzday** [sdispater/orator#102]
when i set the basic usage , i catch the error 
```
Traceback (most recent call last):
  File "C:\Users\fizz_long\AppData\Local\Programs\Python\Python35-32\lib\site-packages\orator\connectors\mysql_connector.py", line 7, in <module>
    import MySQLdb as mysql
ImportError: No module named 'MySQLdb'
```
because my python's version is 3.5.2, so i installed pymysql instead of mysqldb.  
then i find the source of `mysql_connector.py` in line 7 . i change the code
```
try:
        import pymysql as mysql

        # Fix for understanding Pendulum object
        import pymysql.converters
        pymysql.converters.converters[Pendulum] = pymysql.converters.escape_datetime

        from pymysql.cursors import DictCursor as cursor_class
        keys_fix = {}
    except ImportError as e:
        mysql = None
        cursor_class = object
```
in the first , finally like 
```
# -*- coding: utf-8 -*-

import re
from pendulum import Pendulum

try:
    try:
        import pymysql as mysql

        # Fix for understanding Pendulum object
        import pymysql.converters
        pymysql.converters.converters[Pendulum] = pymysql.converters.escape_datetime

        from pymysql.cursors import DictCursor as cursor_class
        keys_fix = {}
    except ImportError as e:
        mysql = None
        cursor_class = object
except ImportError as e:
    import MySQLdb as mysql

    # Fix for understanding Pendulum object
    import MySQLdb.converters
    MySQLdb.converters.conversions[Pendulum] = MySQLdb.converters.DateTime2literal

    from MySQLdb.cursors import DictCursor as cursor_class
    keys_fix = {
        'password': 'passwd',
        'database': 'db'
    }
```
but, it get new error
```
AttributeError: module 'pymysql.converters' has no attribute 'converters'
```
how can i resolve this problem

**Comment by sdispater** [comment]
If I install 'pymysql' I don't have the `ImportError` you encounter, however I still have the second error you have.

Anyway, you can use `MySQLdb` in Python 3 by installing the `mysqlclient` package which is a fork of the original 'MySQLdb'.

I will fix the errors with `pymysql` though, so If you want to use it I'll tell you when it's ready.

**Comment by sdispater** [comment]
OK. Errors for `pymysql` are fixed on the `0.9` branch.

**Comment by fizzday** [comment]
谢谢, 我手动安装了`orator-0.9`版本, 已经没有这个错误了

**Comment by sdispater** [comment]
I will try to release a bugfix release this week.

**Comment by sdispater** [comment]
I just released the bugfix release `0.9.1`.
